src/main/java/net/mcreator/workspace/elements/VariableType.java

```diff
@@ -40,10 +42,21 @@ public String getName() {
 		return name;
 	}
 
-	public int getColor() {
+	public String getColor() {
 		return color;
 	}
 
+	public Color getBlocklyColor() {

```

Comment:
It would be useful some documentations in this class and this method as we are trying to documentate the code.